teeio-validator/teeio_validator/ide_test.c

```diff
@@ -159,12 +159,12 @@ ide_test_group_funcs_t m_group_funcs[] = {
 #define TEST_CLASS_CASE_NAMES "IdeStream,KeyRefresh"
 
 ide_test_case_name_t m_test_case_names[IDE_COMMON_TEST_CASE_NUM] = {
-  {"Query",       "1,2",                  IDE_COMMON_TEST_CASE_QUERY},
-  {"KeyProg",     "1,2,3,4,5,6",          IDE_COMMON_TEST_CASE_KEYPROG},
-  {"KSetGo",      "1,2,3,4",              IDE_COMMON_TEST_CASE_KSETGO},
-  {"KSetStop",    "1,2,3,4",              IDE_COMMON_TEST_CASE_KSETSTOP},
-  {"SpdmSession", "1,2",                  IDE_COMMON_TEST_CASE_SPDMSESSION},
-  {"Test",        TEST_CLASS_CASE_NAMES,  IDE_COMMON_TEST_CASE_TEST}
+  {"Query",       "1,2",                  IDE_COMMON_TEST_CASE_QUERY        , MAX_QUERY_CASE_ID},
+  {"KeyProg",     "1,2,3,4,5,6",          IDE_COMMON_TEST_CASE_KEYPROG      , MAX_KEYPROG_CASE_ID},
```

Comment:
"1,2,3,4,5,6"  and MAX_KEYPROG_CASE_ID  are redundant.
Please keep only one of them.

Reply:
Updated in next version.